x/bounty/keeper/finding.go

```diff
@@ -31,6 +31,9 @@ func (k Keeper) SetFinding(ctx sdk.Context, finding types.Finding) {
 func (k Keeper) GetNextFindingID(ctx sdk.Context) uint64 {
 	store := ctx.KVStore(k.storeKey)
 	Bz := store.Get(types.GetNextFindingIDKey())
+	if Bz == nil {
+		return 1
```

Comment:
I think this is unnecessary and should be initialized at project startup。

Reply:
As we discussed, this is needed.

Reply:
so why is this needed?

Reply:
Like GetProposalID in https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/keeper/proposal.go#L297-L306.   The program will panic if GetNextFindingIDKey is not initialized when be called in ExportGenesis.